docs/en/serverless/elastic-entity-model.mdx

```diff
@@ -0,0 +1,44 @@
+---
+slug: /serverless/observability/elastic-entity-model
+title: Elastic Entity Model
+description: Learn about the model that empowers entity-centric Elastic solution features and workflows.
+tags: [ 'serverless', 'observability', 'overview' ]
+---
+
+import Roles from './partials/roles.mdx'
+
+<p><DocBadge template="technical preview" /></p>
+
+The Elastic Entity Model consists of:
+
+- a data model and related entity indices
+- an Entity Discovery Framework, which consists of [transforms](((ref))/transforms.html) and [Ingest pipelines](((ref))/ingest.html) that read from signal indices and write data to entity indices
+- a set of management APIs that empower entity-centric Elastic solution features and workflows
+
+In the context of Elastic Observability,
+an _entity_ is an object of interest that can be associated with produced telemetry and identified as unique.
+Note that this definition is intentionally closely aligned to the work of the [OpenTelemetry Entities SIG](https://github.com/open-telemetry/oteps/blob/main/text/entities/0256-entities-data-model.md#data-model).
+Examples of entities include (but are not limited to) services, hosts, and containers.
+
+The concept of an entity is important as a means to unify observability signals based on the underlying entity that the signals describe.
+
+## Enable or disable the Elastic Entity Model
+
+<Roles role="Admin" goal="enable or disable the Elastic Entity Model" />
+
+{/* QUESTION: Is this correct? Is an Admin-level role required to do this? */}
```

Comment:
```suggestion
```

Reply:
Removing this comment in preparation for merging, but still need input on the question about whether Admin is required.